common/flatpak-dir.c

```diff
@@ -5363,6 +5364,26 @@ flatpak_dir_deploy (FlatpakDir          *self,
   return TRUE;
 }
 
+static GFile *
+transaction_log_path_for_dir (FlatpakDir *dir)
+{
+  return g_file_get_child (dir->basedir, "transaction-log.json");
+}
+
+
+static gboolean
+log_deploy_install_event (FlatpakDir  *self,
+		          const gchar *ref,
+		          const gchar *origin,
+		          GCancellable *cancellable,
+		          GError      **error)
```

Comment:
Nitpick: Function arguments are not aligned properly.

Reply:
Ugh, auto-tabs strike again :)